src/fdisk.ts

```diff
@@ -0,0 +1,140 @@
+const completionSpec: Fig.Spec = {
+  name: "fdisk",
+  description: "Manipulate disk partition table",
+  options: [
+    {
+      name: ["--help", "-h"],
+      description: "Show help for fdisk",
+    },
+    {
+      name: ["--version", "-V"],
+      description: "Show version for lsblk",
+    },
+    {
+      name: ["--sector-size", "-b"],
+      description: "Specify the sector size of the disk",
+      args: {
+        name: "mode",
+        description: "Valid values are 512, 1024, 2048, and 4096",
+      },
+    },
+    {
+      name: ["--protect-boot", "-B"],
+      description:
+        "Don't erase the beginning of the first disk sector when creating a new disk label",
+    },
+    {
+      name: ["--compatibility", "-c"],
+      description: "Specify the compatibility mode, 'dos' or 'nondos'",
+      args: {
+        name: "mode",
+        isOptional: true,
+      },
+    },
+    {
+      name: ["--color", "-L"],
+      description: "Colorize the output",
+      args: {
+        name: "when",
+        isOptional: true,
+      },
+    },
+    {
+      name: ["--list", "-l"],
+      description:
+        "List the partition tables for the specified devices and then exit",
+    },
+    {
+      name: ["--list-details", "-x"],
+      description: "Like --list, but provides more details",
+    },
+    {
+      name: "--lock",
+      description: "Use exclusive BSD lock for device or file it operates",
+      args: {
+        name: "mode",
+        description:
+          "Optional argument mode can be yes, no (or 1 and 0) or nonblock",
+        isOptional: true,
+      },
```

Comment:
Can have suggestions! I like to add the more verbose ones if equivalent because it's easier to read at a glance.
```suggestion
      args: {
        name: "mode",
        description:
          "Optional argument mode can be yes, no (or 1 and 0) or nonblock",
        isOptional: true,
        suggestions: ["yes", "no", "nonblock"],
      },
```